doc/classes/Joint2D.xml

```diff
@@ -74,12 +74,18 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_bias" getter="get_bias">
+			Currently only used in [PinJoint2D].
+			When [member node_a] and [member node_b] move in different directions, this value defines how much they get pulled back into the same positional relation to each other.
```

Comment:
Merge the 2 paragraphs. "positional relation to each other" lacks clarity. Another passive. You have a link to this variable.
`When [member node_a] and [member node_b] move in different directions the [code]bias[/code] controls how fast the joint pulls them back to their original position. The lower the [code]bias[/code] the more the two bodies can pull on the joint. Default value: ...`